apps/arweave/src/ar_nonce_limiter.erl

```diff
@@ -417,34 +417,21 @@ init([]) ->
 			_ ->
 				#state{}
 		end,
+	case ar_config:use_remote_vdf_server() and not ar_config:compute_own_vdf() of
+		true ->
+			gen_server:cast(?MODULE, check_external_vdf_server_input);
+		false ->
+			ok
+	end,
 	State2 =
-		case ar_config:use_remote_vdf_server() of
+		case ar_config:compute_own_vdf() of
 			false ->
-				State;
+				State#state{ autocompute = false };
 			true ->
-				resolve_remote_server_raw_peers(),
-				gen_server:cast(?MODULE, check_external_vdf_server_input),
-				State#state{ autocompute = false }
+				State
```

Comment:
I think we can replace this with:

```
State2 = State#state{ autocompute = ar_config:compute_own_vdf() },
```
`autocompute` defaults to `true` and is only set to false if compute_own_vdf() is false. So I think ogically it's the same.